moped-editor/src/components/GridTable/Filters.js

```diff
@@ -132,20 +133,20 @@ const generateEmptyField = (uuid) => {
 const Filters = ({
   filters,
   setFilters,
-  filterQuery,
-  history,
   handleAdvancedSearchClose,
   filtersConfig,
   setSearchFieldValue,
   setSearchTerm,
+  isOr,
+  setIsOr,
 }) => {
   /**
    * The styling of the search bar
    * @constant
    * @default
    */
   const classes = useStyles();
-  const queryPath = useLocation().pathname;
+  let [, setSearchParams] = useSearchParams();
```

Comment:
TIL—i did not know you can destructure like this!

Reply:
same!! I was reading MDN and testing stuff.